File: src/main/java/me/ctknight/uploadmanager/util/LogUtils.java
/*
 * Copyright (c) 2016.  All rights reserved. Lai Jiewen <alanljw12345@gmail,com
 */

package me.ctknight.uploadmanager.util;

public class LogUtils {

    private LogUtils() {
    }

    public static String makeTag(Class tClass) {
        return tClass.getSimpleName();
    }
}

File: lib/src/main/java/me/ctknight/uploadmanager/util/LogUtils.kt
/*
 * Copyright (c) 2018. All rights reserved. Lai Jiewen <alanljw12345@gmail.com>
 */

package me.ctknight.uploadmanager.util

internal object LogUtils {
  internal inline fun <reified T> makeTag(): String {
    return T::class.java.simpleName
  }
}
